### DISCUSSION TITLE
The resource checks are not checking

### DISCUSSION THREAD

**Discussion by skreddy6673** [concourse/concourse#5507] [General]
The time interval check you will see on the pipeline page seems to be not performing.
I've set my interval of a time resource to every one minute, which means the time shown on the resource page should not exceed 1 minute, but my resource check keeps on increasing the value and checking at the Global value (24 Hr) set in Helm Deployment chart.
![image](https://user-images.githubusercontent.com/27447138/80380593-c79d2980-886d-11ea-91d0-b2292c55a9b6.png) This is what i'm referring to.
This is not happening with all the pipelines in my concourse Instance. Some are working just fine.

Somehow deleting the resource type and adding it back after some time seems to be working for now.
Concourse: v5.7.1 and v5.4.1

**Comment by vito** [comment]
Have you set [`check_every: 1m`](https://concourse-ci.org/resources.html#schema.resource.check_every)? Concourse doesn't know anything about the `source: {interval: 1m}` portion, so you'll need to tell it to override the default checking interval (which it sounds like you've set to `24h`).

**Reply by skreddy6673** [reply]
Yes, i have `check_every set to 1m`
